src/Components/Publishing/Header/FeatureHeader.tsx

```diff
@@ -350,388 +387,8 @@ const FeatureHeaderContainer = DivWithProps`
       ${HeaderText} {
         padding: 20px;
       }
-    `}
+    `};
   }
 `
 
 export const FeatureHeader = FeatureHeaderComponent
-
-// FIXME: Reeimplment below refactor once the heat is off....
```

Comment:
I think the below can probably go since whatever should likely conform to a grid layout

Reply:
@damassi what do you mean by this? Go as in be used?

Reply:
Oh haha, no go meaning delete!